README.md

```diff
@@ -177,6 +177,7 @@ need to be evaluated before non-fitting events can be dropped.
 | `isValidWindowsTimeZoneId`                      | `$timeZone`                                                                                   | `protected` | Checks if a time zone is a recognised Windows (non-CLDR) time zone                                                                                               |
 | `isValidTimeZoneId`                             | `$timeZone`                                                                                   | `protected` | Checks if a time zone is valid (IANA, CLDR, or Windows)                                                                                                          |
 | `keyValueFromString`                            | `$text`                                                                                       | `protected` | Gets the key value pair from an iCal string                                                                                                                      |
```

Comment:
`keyValueFromString` is being made `public` within this PR, but is still documented as `protected` here.